.github/workflows/deploy.yml

```diff
@@ -9,10 +9,15 @@ on:
         required: true
         options:
           - S3
+          - Payara
       environment:
         description: Environment to run the deployment against
         type: environment
         required: true
+      basepath:
+        description: 'Server basepath, without slashes, to serve the application (For example: spa). This is only for Payara deployment, and should be left blank if a basepath is not needed.'
```

Comment:
```suggestion
        description: 'Server basepath, without slashes or any spaces or punctuation, to serve the application (For example: "spa"). This is only for Payara deployment, and should be left blank if a basepath is not needed.'
```

Reply:
This is a minor point that I'll let @kcondon consider in QA. Not a blocker at all. I just think it reads a little better to have "spa" in quotes and I like giving a bit more guidance (even if it's stricter than necessary) on what to type in the basepath field.